setup.sh

```diff
@@ -10,6 +10,10 @@ if [ $EUID -ne 0 ]; then
   exit 1
 fi
 
+if git submodule status --recursive | grep -q "^-"; then
+  sudo -u $SUDO_USER git submodule update --init
```

Comment:
Why are you using sudo here?  It shouldn't be necessary.  Only the installer needs sudo.